modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -469,7 +469,10 @@ protected ImmutableMap.Builder<String, Lambda> addMustacheLambdas() {
                 .put("indented", new IndentedLambda())
                 .put("indented_8", new IndentedLambda(8, " ", false, false))
                 .put("indented_12", new IndentedLambda(12, " ", false, false))
-                .put("indented_16", new IndentedLambda(16, " ", false, false));
+                .put("indented_16", new IndentedLambda(16, " ", false, false))
+                .put("tidy", new TidyLambda())
+                .put("tidyForDocs", new TidyLambda(true))
```

Comment:
what about using just lowercase following the convention in the other lambdas? e.g. tidydocs

Reply:
I'll rename it later before 7.4.0 release.

Reply:
Hi! I am ok with that! Thank you!